cmd/find.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/puppetlabs/wash/api/client"
+	cmdfind "github.com/puppetlabs/wash/cmd/find"
+	cmdutil "github.com/puppetlabs/wash/cmd/util"
+	"github.com/puppetlabs/wash/config"
+	"github.com/spf13/cobra"
+)
+
+func findCommand() *cobra.Command {
+	findCmd := &cobra.Command{
+		Use: "find <path> [expression]",
+		// TODO: More detailed usage. Will need to use custom help text in order to
+		// properly enumerate all the primaries.
+		Short: "Finds stuff",
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
Improvements: it'd be nice to be able to say `wish find` and have that be equivalent to `wash find .`.

Also like to see it work at the plugin root level, and maybe even outside of the wash directory hierarchy.

Reply:
I'll add those as TODOs